src/components/EntityInfo/EntityHeader.vue

```diff
@@ -157,4 +157,10 @@ export default class EntityHeader extends Vue {
 #active-util {
   border-left: 1px solid $gray3;
 }
+
+// Style for badges
+:deep(.v-chip.v-size--small) {
+  height: 19px;
```

Comment:
Why 19?

Reply:
From Janis' comment:
![janis](https://github.com/bcgov/business-filings-ui/assets/122301442/9e347e25-0385-471e-a983-22807879e5b3)

Reply:
So that's 1.1875 rem then.

Just kidding. This is fine. Weird, but fine.